sections/docker.zsh

```diff
@@ -24,13 +24,33 @@ spaceship_docker() {
 
   spaceship::exists docker || return
 
+  # Better support for docker environment vars: https://docs.docker.com/compose/reference/envvars/
+  # Use COMPOSE_PATH_SEPARATOR or colon as default
+  local separator=":"
+  [[ ! -z $COMPOSE_PATH_SEPARATOR ]] && separator=$COMPOSE_PATH_SEPARATOR
+
+  # COMPOSE_FILE may have several filenames separated by colon, test all of them
+  local filenames=("${(@ps/$separator/)COMPOSE_FILE}")
+
+  local compose_exists
+  for filename in $filenames; do
+    if [[ ! -f $filename ]]; then
+      compose_exists=false
+      break
+    fi
+    compose_exists=true
+  done
+
   # Show Docker status only for Docker-specific folders
-  [[ -f $COMPOSE_FILE || -f Dockerfile || -f docker-compose.yml ]] || return
+  [[ "$compose_exists" == true || -f Dockerfile || -f docker-compose.yml ]] || return
 
   # if docker daemon isn't running you'll get an error saying it can't connect
   local docker_version=$(docker version -f "{{.Server.Version}}" 2>/dev/null)
   [[ -z $docker_version ]] && return
 
+  # Support for $COMPOSE_PROJECT_NAME (docker env vars)
+  [[ ! -z "$COMPOSE_PROJECT_NAME" ]] && docker_version+=" ($COMPOSE_PROJECT_NAME)"
```

Comment:
Let's not include `COMPOSE_PROJECT_NAME` here and focus this PR on `COMPOSE_FILE`.  We can discuss about `COMPOSE_PROJECT_NAME` in new issue/PR.

Reply:
All right!